pkgs/top-level/php-packages.nix

```diff
@@ -663,7 +663,7 @@ in {
           # Some tests are causing issues in the Darwin sandbox with issues
           # such as
           #   Unknown: php_network_getaddresses: getaddrinfo for localhost failed: nodename nor servname provided
-          doCheck = !stdenv.hostPlatform.isDarwin;
+          doCheck = !stdenv.hostPlatform.isDarwin && lib.versionOlder php.version "8.4";
```

Comment:
See the commit message for the rationale. If somebody has a better solution, I'm all for it.